contrib/docker/multi-stage-with-bundle/Dockerfile

```diff
@@ -0,0 +1,42 @@
+# Stage 1 - Create yarn install skeleton layer
+FROM node:14-buster AS packages
+
+WORKDIR /app
+COPY package.json yarn.lock ./
+
+# Uncomment this line if building a non create-app version
+# COPY packages packages
```

Comment:
Seems inverted, packages is always there, plugins may not be

Reply:
derp